README.md

```diff
@@ -8,4 +8,4 @@ Install [Coder](https://github.com/coder/coder) to your preferred cloud with a p
 | <img src="./assets/fly.io.svg" alt="Fly.io Logo" width="100" height="100" />                           | Fly.io                | Live ✅     | [Blog: Run Coder on Fly.io](https://coder.com/blog/remote-developer-environments-on-fly-io)            | [Deploy Coder on FLy.io](https://coder.com/blog/remote-developer-environments-on-fly-io)                                                   |
 | <img src="https://railway.app/brand/logo-light.svg" alt="Railway.app Logo" width="100" height="100" /> | Railway.app           | Live ✅     |                                                                                                        | [![Deploy on Railway](https://railway.app/button.svg)](https://railway.app/template/cUQ8_P?referralCode=tfH8Uw)                            |
 | <img src="./assets/eks.svg" alt="AWS EKS Logo" width="100" height="100" />                             | AWS EKS               | In progress | [Docs: Coder on Kubernetes](https://coder.com/docs/v2/latest/install/kubernetes)                       | [Deploy from AWS Marketplace](https://example.com)                                                                                         |
-| <img src="./assets/gce.svg" alt="Google Compute Engine logo" width="100" height="100" />               | Google Compute Engine | Considering | [Quickstart: Google Compute Engine](https://coder.com/docs/v2/latest/quickstart/google-cloud-platform) |                                                                                                                                            |
+| <img src="./assets/gce.svg" alt="Google Compute Engine logo" width="100" height="100" />               | Google Compute Engine | Considering | [Guide: Google Compute Engine](https://coder.com/docs/v2/latest/platforms/google-cloud-platform) |                                                                                                                                            |
```

Comment:
https://coder.com/docs/v2/latest/platforms/google-cloud-platform
this is 404

Reply:
It won't 404 as soon as https://github.com/coder/coder/issues/6971 is merged